src/app/(home)/newPassword/page.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import { LetterIcon } from "../../../assets/script/changePassword/LetterIcon";
+
+import { PaperAirLineIcon } from "../../../assets/script/changePassword/PaperAirLineIcon";
+import { OpenLetterIcon } from "../../../assets/script/changePassword/OpenLetterIcon";
+import { ArrowsIcon } from "../../../assets/script/changePassword/ArrowsIcon";
+import { QuestionIcon } from "../../../assets/script/changePassword/QuestionIcon";
+import { EmailIcon } from "../../../assets/script/changePassword/EmailIcon";
+
+import { OvalIcon } from "../../../assets/script/changePassword/OvalIcon";
+import { ManIcon } from "../../../assets/script/changePassword/ManIcon";
+
+import { INewPassword } from "../../../types/pages/Password";
+import { NewPasswordModal } from "../../../components/appResponse/newPasswordModal/newPasswordModal";
+
+import style from "./newPassword.module.scss";
+
+
+export default function NewPassword() {
+
+	const [isNewPasswordModalShown, setIsNewPasswordModalShown] = useState(false);
+	
+	const {
+		register,
+		handleSubmit,
+		reset,
+		formState: { errors },
+	} = useForm<INewPassword>({ mode: "onBlur" });
+
+	const onSubmit = (data: INewPassword) => {
+		alert(JSON.stringify(data));
+		handleNewPasswordModal();
+		reset();
+	};
+
+	const handleNewPasswordModal = () => {
+		setIsNewPasswordModalShown(true)
+	}
+
+	return (
+		<div className={style.newPasswordWrap}>
+			<div className={style.newPasswordContainer}>
+				<OvalIcon classNames={style.ovalIcon} />
+				<QuestionIcon classNames={style.questionIcon} />
+				<EmailIcon classNames={style.emailIcon} />
+				<ArrowsIcon classNames={style.arrowsIcon} />
+				<LetterIcon classNames={style.letterIcon} />
+				<OpenLetterIcon classNames={style.openLetterIcon} />
+				<ManIcon classNames={style.manIcon} />
+				<PaperAirLineIcon classNames={style.paperAirLineIcon} />
+				<OpenLetterIcon classNames={style.secondOpenLetterIcon} />
+				<div className={style.newPasswordContainer__modal}>
+					<div className={style.newPasswordContainer__modal__content}>
+						<h1 className={style.newPasswordContainer__form__title}>Восстановление пароля</h1>
+						<form onSubmit={handleSubmit(onSubmit)}>
+						{isNewPasswordModalShown && <NewPasswordModal/>}
+							<label htmlFor="enterEmail" className={style.label}>Введите почту</label>
```

Comment:
formWrap__emailTitle

Crating class names using the official HTML tag name is approach that worsens semantics. label is a tag for creating **text** above input elements